app/Entities/Place/RatingAverage.php

```diff
@@ -4,12 +4,31 @@
 
 class RatingAverage
 {
-    public $placeId;
-    public $avgRating;
+    private $placeId;
+    private $avgRating;
 
     public function __construct(int $placeId, float $avgRating)
     {
         $this->placeId = $placeId;
         $this->avgRating = $avgRating;
     }
+
+    public function getPlaceId(): int
+    {
+        return $this->placeId;
+    }
+
+    public function getAvgRating(): float
+    {
+        return $this->avgRating;
+    }
+
+    public function __get($name)
```

Comment:
Remove, not necessary, you already have getAvgRating(), getPlaceId() to call in GetUserCheckInCollectionPresenter